drivers/SmartThings/samsung-audio/src/disco.lua

```diff
@@ -145,6 +155,7 @@ function Disco.find(deviceid, callback)
 
         if id == deviceid then
```

Comment:
Related to another comment:

```suggestion
        if deviceid and id == deviceid then
```

Reply:
Done